templates/observability/what-is-observability.html

```diff
@@ -173,7 +172,7 @@ <h3 class="p-heading--4">Making the most out of your metrics, logs, distributed
             <p>
                 {{ image (
                     url="https://assets.ubuntu.com/v1/cf7d3849-telemetry-map-v2.svg",
-                    alt="",
+                    alt="Logs, metrics, distributed traces, production profiling, synthetic monitoring and end-user monitoring all contribute to observability, documenting different aspects of the performance of a system",
```

Comment:
Image alt text are not a way to include keywords but should describe the image to someone using a screen reader. Do you think this effectively describes that image to that audience?

Reply:
I think it describes it a bit. Could be of course longer: that image is making _a lot_ of work. What is an acceptable length for an `alt` tag?

@caldav what's your take on this matter?

Reply:
I would reword the following way to make it more descriptive: "Diagram of observability components of a system: logs, metrics, distributed traces, production profiling, synthetic and end-user monitoring". If you were using a screen reader, you'd want the big concept first, then the details. The rough limit is 15 words (~130 characters).

Reply:
I kept a little more info than that, there is no write-up yet in the page about the correlation among different types of telemetry.